Applications/Spire/Include/Spire/Ui/ListView.hpp

```diff
@@ -227,10 +227,14 @@ namespace Styles {
       int get_index_by_value(const std::any& value) const;
       QLayout* get_layout();
       QLayoutItem* get_column_or_row(int index);
+      bool is_column_or_row_disabled(QLayout* target, int begin, int end);
       void select_item(const boost::optional<std::any>& selection);
-      void cross_move(bool is_next);
-      int move_next();
-      int move_previous();
+      void cross_move(int position, int index, bool is_next, bool track_move);
```

Comment:
Can track_move be removed? I see it passed to multiple functions but I can't see where it's used.

Reply:
You are right. It is redundant and should be removed.